lib/Wikia/src/Service/User/Preferences/PreferenceModule.php

```diff
@@ -10,6 +10,8 @@
 use Wikia\Persistence\User\Preferences\PreferencePersistenceSwaggerService;
 
 class PreferenceModule implements Module {
+	const SWAGGER_SERVICE_RAMP_USAGE = 0;
```

Comment:
I thought about that, but I think it'd be nice to have a config in MediaWiki as close to the thing that uses it since there are a million configs in the config repo. Also, this config should not be used by anything other than this class, so i don't think we need to introduce yet-another `$wg...` variable. It'd be confusing to have both `$wgPreferencesUseService` and (potentially) `$wgPreferencesUseSwaggerService` in the config repo.